docs/requirements.txt

```diff
@@ -1,3 +1,5 @@
+sphinx=2.2.1
```

Comment:
This file can be deleted, see comments in `environment.yaml` and `.readthedocs.yaml`